assets/src/application/services/hooks/useMemoStringify.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+import { is } from 'ramda';
+
+const useMemoStringify = (data: any, deps?: any[]): any => {
```

Comment:
I tried to pass the second argument as an object with `deps` property, but still think that is't maybe better to split the logic here into 2 hooks

Reply:
also those `any` types also hinders me a bit;
I have mixed feelings about this hook

Reply:
it doesn't solve a lot whole of stuff, but it makes the end components a bit cleaner w/o that `stringify` or `join` logic